common/djangoapps/student/tests/tests.py

```diff
@@ -219,7 +219,10 @@ def _check_verification_status_on(self, mode, value):
             attempt.approve()
 
         response = self.client.get(reverse('dashboard'))
-        self.assertContains(response, "class=\"course {0}\"".format(mode))
+        if mode in ['professional', 'no-id-professional']:
+            self.assertContains(response, "class=\"course professional")
```

Comment:
Is it necessary to escape this quote in the middle of the string? It looks left over from copying the string below, `"class=\"course {0}\""`.